### ISSUE TITLE
Can request a vendor ID for Mesa-IR to SPIR-V?

### ISSUE THREAD

**Issue by metora** [KhronosGroup/SPIRV-Headers#40]
Hello

I have a Mesa-IR to SPIR-V Translator.
My Github is https://github.com/metora/MesaGLSLCompiler.
It is a prototype now.
It can input a GLSL 1.00~4.50 Shader (Mesa GLSL Compiler Based).
But It supported one uniform buffer only now.

Thanks.

**Comment by johnkslang** [comment]
Are you wanting a SPIR-V Tool ID?  See the first section of [SPIR-V registry](https://github.com/KhronosGroup/SPIRV-Headers/blob/master/include/spirv/spir-v.xml). Can you post a PR suggesting the line you'd like to add?  Probably editing the following line:

```
 <unused start="16" end="0xFFFF" comment="Tool ID range reservable for future use by vendors"/>
```

**Comment by metora** [comment]
Yes, I need a Tool ID.

<id value="16" vendor="X-LEGEND" tool="Mesa-IR/SPIR-V Translator" comment="github metora/MesaGLSLCompiler"/>

Is it right?

Thanks.

**Comment by johnkslang** [comment]
Yes, something like that. You'll want to duplicate the line to preserve the next spot.

However, followup with a real PR: It's a much better place to look at code changes and discuss and fix. The PR does not have to be correct the first time, and is easy to modify until we agree.

**Comment by johnkslang** [comment]
Resolved by #41.
